src/Microsoft.Identity.Client/Internal/Requests/SilentRequest.cs

```diff
@@ -65,22 +65,28 @@ protected override void SetAdditionalRequestParameters(OAuth2Client client)
             if (!ForceRefresh)
             {
                 AccessTokenItem
-                     = TokenCache.FindAccessToken(AuthenticationRequestParameters);
+                    = TokenCache.FindAccessToken(AuthenticationRequestParameters);
```

Comment:
if force refresh is true and authority is not passed, you skip the access token lookup, what is the authority you're gonna use?